app/views/api/v1/transactions/show.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.extract! @transaction, :id, :invoice_id,
```

Comment:
@pwentz I think I prefer these all on their own lines, but if you strongly prefer it this way stylistically that's good with me.